src/NuGet.Core/NuGet.Commands/RestoreCommand/RequestFactory/RestoreArgs.cs

```diff
@@ -137,7 +137,7 @@ private List<SourceRepository> GetEffectiveSourcesCore(ISettings settings, IList
                 //DGSpecSources should always match the Sources
                 if (!sourceObjects.ContainsKey(sourceUri))
                 {
-                    Log.LogDebug($"{sourceUri} is in the RestoreArgs Sources but in the passed in dgSpecSources");
+                    Log.LogDebug($"{sourceUri} is in the RestoreArgs Sources but not in the passed in dgSpecSources");
```

Comment:
add this to strings.resx so it can be localized unless we never localize debug strings.